en/06_Upgrading/07_Deprecations.md

```diff
@@ -73,10 +73,13 @@ SilverStripe\Core\Injector\Injector:
 > ```bash
 > SS_ERROR_LOG="./silverstripe.log"
 > ```
->
-> You don't need any of the YAML configuration above if you are using the `SS_ERROR_LOG` environment variable - but you can use a combination of the environment variable and YAML configuration if you want to configure multiple error log files.
->
-> You will also need to make sure the user running the PHP process has write access to the log file, wherever you choose to put it.
+
+You don't need any of the YAML configuration above if you are using the `SS_ERROR_LOG` environment variable - but you can use a combination of the environment variable and YAML configuration if you want to configure multiple error log files.
+
+You will also need to make sure the user running the PHP process has write access to the log file, wherever you choose to put it.
```

Comment:
I've moved this bit out of the rather large !WARNING block above mainly to prevent linting errors because of the newly added !NOTE block below

Reply:
Probably better this way anyway tbh.